tests/spec/cmdline_spec.rb

```diff
@@ -99,16 +99,40 @@ def self.pr_number
       expect(result).to be true
     end
   end
+end
+
+# secondary --changed_since
+describe 'cmdline changed-since' do
+  before(:each) do
+    @gitarrorepo = GIT_REPO
+    @rgit = GitRemoteOperations.new(@gitarrorepo)
+    @pr = @rgit.first_pr_open
+    @test = GitarroTestingCmdLine.new(@gitarrorepo)
+    # commit status
+    @comm_st = @rgit.commit_status(@pr)
+  end
+
+  describe '.changed_since_60' do
+    it "gitarro should see PR ##{PR_NUMBER} changed in the last 60 seconds" do
+      context = 'changed-since-60'
+      pr = @rgit.pr_by_number(PR_NUMBER)
+      comment = @rgit.create_comment(pr, "gitarro rerun #{context} !!!")
+      result, output = @test.changed_since(@comm_st, 60, context, context)
+      @rgit.delete_c(comment.id)
+      expect(result).to be true
+      expect(output[0]).to match(/^\[PRS=TRUE\].*/)
+    end
+  end
 
-  describe '.changed_since' do
-    it 'gitarro should see at least PR #30 changed in the last 60 seconds' do
-      context = 'changed-since'
-      desc = 'changed-since'
+  describe '.changed_since_zero' do
+    it 'gitarro should not see any PRs' do
+      context = 'changed-since-zero'
       pr = @rgit.pr_by_number(PR_NUMBER)
       comment = @rgit.create_comment(pr, "gitarro rerun #{context} !!!")
-      result = @test.changed_since_should_find(@comm_st, 60, context, desc)
+      result, output = @test.changed_since(@comm_st, 0, context, context)
       @rgit.delete_c(comment.id)
       expect(result).to be true
+      expect(output[0]).to match(/^\[PRS=FALSE\].*/)
```

Comment:
@juliogonzalez  since we have these 2 tests, i think we don't need the unit-tests anymore.
#74 ( imho this is fixed) with this 2 tests ( we need but to remove the units

Reply:
As far as I can remember, there is one unit that we still need.

I will check.

Reply:
There was a test to check if everything was fine when --changed_since was not present.

I think I should move it to be an acceptance test, or change the basic_test to look for [PRS=true] as well.

Any thoughts?

Reply:
@juliogonzalez  imho we should have 3 tests;

1. changed_since = 0
2. changed_since = 60
3. changed_since = -1 ( this test is GRATIS) so we test this when the option is not set.
   Imho adding the extra check for grepping the output in basic-test is adding more complexity and dependance .
Imagine we remove this output , and then the basic test will fail.
Imho we need  a 3rd test that is called `changed_since_not_set_but_check_output` , with is not adding more dependency to other tests ( for debugging it will be more difficult)

Reply:
Well, I do not think we should force -1, since that way ware by passing the command line default value when --change_since is not available.

So it should be a test called ```changed_since_not_present``` sending -1 to ```changed_since``` at test_lib.rb and then that function not adding --change_since.

Changes are almost ready.